provision.sh

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env bash
+set -e
+
+function wrong_args() {
+  echo >&2 "Wrong number of arguments! Usage:"
+  echo >&2 "  $(basename "$0")"
+  echo >&2 "  $(basename "$0") region-name country-code-list [osm-file-download-url]"
+  echo >&2 "Example:"
+  echo >&2 "  $(basename "$0") belgium BE http://download.geofabrik.de/europe/belgium-latest.osm.pbf"
+  exit 1
+}
+
+function usage() {
+  echo "Usage:"
+  echo "  $(basename "$0")"
+  echo "  $(basename "$0") region-name country-code-list [osm-file-download-url]"
```

Comment:
The parameter region-name is used in different ways. In example 2, there is exact name of the osm file "belgium-latest.osm.pbf", while in example 3, there is just "belgium". Do we use it in a different context or are just the provided examples misleading?